gemelli/matrix_completion.py

```diff
@@ -0,0 +1,169 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2019--, gemelli development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+from gemelli.optspace import OptSpace
+from .base import _BaseImpute
+from scipy.spatial import distance
+
+
+class MatrixCompletion(_BaseImpute):
```

Comment:
I am going to assume this is copy and pasted from `deicode`

Reply:
Correct.